### PR TITLE
Cross browser config in nightly

### PR DESCRIPTION
**Pull request by nraghu32**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ ] No

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -36,8 +41,8 @@ env.IDAM_URL = 'https://idam-api.aat.platform.hmcts.net'
 
 withPipeline(type, product, component) {
   enableAksStagingDeployment()
+  enableCrossBrowserTest(['chrome', 'firefox'])
```
Comment (abidfs) [inline Jenkinsfile_CNP:44]: why are we enabling cross browser tests in all pipeline run?

Comment (nraghu32) [reply Jenkinsfile_CNP]: Its for test purposes removed now

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -66,5 +66,12 @@ withPipeline(type, product, component) {
   before('functionalTest:aat') {
     yarnBuilder.yarn('test:functional:skip')
   }
+  before('crossBrowserTest') {
+    env.TEST_RETRIES = 5
+  }
+
+  after('crossBrowserTest') {
+    steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'output/**/*'
+  }
```
Comment (abidfs) [inline Jenkinsfile_CNP:75]: I think these are not required

Comment (nraghu32) [reply Jenkinsfile_CNP:75]: Removed

**Review by abidfs — APPROVED** [review]

**Pull request merged**